packages/mobile/src/screens/notifications-screen/Notifications/RemixCosignNotification.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback } from 'react'
+
+import {
+  getNotificationEntities,
+  getNotificationUser
+} from 'audius-client/src/common/store/notifications/selectors'
+import { RemixCosign } from 'audius-client/src/common/store/notifications/types'
+import { isEqual } from 'lodash'
+import { View } from 'react-native'
+
+import IconRemix from 'app/assets/images/iconRemix.svg'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { getTrackRoute } from 'app/utils/routes'
+
+import {
+  NotificationHeader,
+  NotificationText,
+  NotificationTile,
+  NotificationTitle,
+  EntityLink,
+  UserNameLink,
+  ProfilePicture
+} from '../Notification'
+import { useDrawerNavigation } from '../useDrawerNavigation'
+
+const messages = {
+  title: 'Remix Co-sign',
+  cosign: 'Co-signed your Remix of'
+}
+
+type RemixCosignNotificationProps = {
+  notification: RemixCosign
+}
+
+export const RemixCosignNotification = (
+  props: RemixCosignNotificationProps
+) => {
+  const { notification } = props
+  const navigation = useDrawerNavigation()
+  const { childTrackId } = notification
+  const user = useSelectorWeb(state => getNotificationUser(state, notification))
+  const tracks = useSelectorWeb(
+    state => getNotificationEntities(state, notification),
+    isEqual
+  )
+
+  const childTrack = tracks.find(({ track_id }) => track_id === childTrackId)
+  const parentTrack = tracks.find(({ track_id }) => track_id === childTrackId)
```

Comment:
Looks like we're searching for `childTrackId` in both of them?

Reply:
oh damn great catch